backend/src/main/java/ch/sectioninformatique/template/auth/OAuth2Controller.java

```diff
@@ -1,100 +0,0 @@
-package ch.sectioninformatique.template.auth;
```

Comment:
The OAuth2 authentication API could be included in AuthController (see comment in AuthController).